interface_frontend.py

```diff
@@ -158,20 +163,21 @@ def draw_interface(interface_state, window):
     # Interface background
     interface_sprite.draw()
 
-    if interface_state.robot_data:
+    if interface_state.robot:
         # Robot
+        my_robot_sprite.img = loaded_robots_images[interface_state.robot.name]
```

Comment:
```suggestion
        my_robot_sprite.image = loaded_robots_images[interface_state.robot.name]
```

Omlouvám se. V taháku je chyba.